CONTRIBUTE.md

```diff
@@ -57,6 +57,39 @@ If anything shows up, then the port is still locked.
 -   `_settingsViewer` - a webapp that provides a visual interface to the configuration files.
 -   `_integrationTests` - a test suite that launches the extension in VS Code.
 
+## Adding configurations
+
+1. Edit `CSpellSettingsPackageProperties` in _server/src/config/cspellConfig.ts_ to add your configuration field, e.g.
```

Comment:
Configuration settings go in `SpellCheckerSettings`. The `CSpellSettingsPackageProperties` are where types imported from CSpell are added and annotated.

A link to `server/src/config/cspellConfig.ts`:
```
[server/src/config/cspellConfig.ts](./packages/_server/src/config/cspellConfig.ts)
``` 
Might be better.

Reply:
> Configuration settings go in `SpellCheckerSettings`. The `CSpellSettingsPackageProperties` are where types imported from CSpell are added and annotated.

Not sure I understand. Did I added the `suggestionMenuType` in the right place? I added it only in `CSpellSettingsPackageProperties`. Should I make a another PR fixing it?

> A link to server/src/config/cspellConfig.ts:
>
> [server/src/config/cspellConfig.ts](./packages/_server/src/config/cspellConfig.ts)
>
> Might be better.

that's true. updated